server/models/Transaction.js

```diff
@@ -397,6 +397,11 @@ export default (Sequelize, DataTypes) => {
     transaction.TransactionGroup = uuidv4();
     transaction.hostCurrencyFxRate = transaction.hostCurrencyFxRate || 1;
 
+    if (!isUndefined(transaction.amountInHostCurrency)) {
```

Comment:
@eloyekunle do you know what was the breaking change in sequelize?

@Betree can you review this?

Reply:
It seems Sequelize was used to coercing Double values to fit into Integer columns until v5.

The `amountInHostCurrency` column is an integer column, but we were passing in floats in some cases so it was returning errors.